### PR TITLE
Updating documentation to include helm 2 to helm 3 chart migration

### PR DESCRIPTION
**Pull request by mrepop**
Updated with process for helm 2 to helm 3 migration and conversion

### CODE REVIEW

File: docs/docsource/installation/upgrade/upgrade_zenko.rst [hunk 1]
```diff
@@ -87,3 +87,34 @@ To upgrade Zenko:
       Expect the cluster to take a few minutes to stabilize. You may see 
       CrashLoopBackoff errors. This is normal, expected behavior.
 
+#. With the introduction of Helm 3 support of Helm 2 charts are still 
+   supported. Helm 3 charts introduces improved functionality with support for 
+   existing Helm 2 charts and in place upgrades and conversions.
```
Comment (wabernat) [inline docs/docsource/installation/upgrade/upgrade_zenko.rst]:
```suggestion
With the introduction of Helm 3, Helm 2 charts are still 
supported. Helm 3 charts introduce improved functionality, and support
existing Helm 2 charts and in-place upgrades and conversion.
```

File: docs/docsource/installation/upgrade/upgrade_zenko.rst [hunk 2]
```diff
@@ -87,3 +87,34 @@ To upgrade Zenko:
...
+ 
+   To upgrade existing Helm 2 charts to Helm 3 you can install the 2to3
+   plugin and use the following steps to migrate and cleanup your
```
Comment (wabernat) [inline docs/docsource/installation/upgrade/upgrade_zenko.rst]:
```suggestion
   To upgrade existing Helm 2 charts to Helm 3, install the 2to3
   plugin and use the following steps to migrate and clean up your 
```

File: docs/docsource/installation/upgrade/upgrade_zenko.rst [hunk 3]
```diff
@@ -87,3 +87,34 @@ To upgrade Zenko:
...
 
+#. With the introduction of Helm 3 support of Helm 2 charts are still 
+   supported. Helm 3 charts introduces improved functionality with support for 
+   existing Helm 2 charts and in place upgrades and conversions.
+ 
+   To upgrade existing Helm 2 charts to Helm 3 you can install the 2to3
+   plugin and use the following steps to migrate and cleanup your 
+   Helm 2 configuration, releases, and Tiller deployment. 
+
+   First install the 2to3 plugin for Helm 3.
```
Comment (wabernat) [inline docs/docsource/installation/upgrade/upgrade_zenko.rst]:
```suggestion
#. Install the 2to3 plugin for Helm 3.
```

File: docs/docsource/installation/upgrade/upgrade_zenko.rst [hunk 4]
```diff
@@ -87,3 +87,34 @@ To upgrade Zenko:
...
+  
+  ::
+
+     $ helm3 plugin install https://github.com/helm/helm-2to3
+
+  Then update the existing Helm 2 configuration.  As previously noted always 
+  test with the --dry-run option before migrating or updating production 
+  systems.
```
Comment (wabernat) [inline docs/docsource/installation/upgrade/upgrade_zenko.rst]:
```suggestion
#. Update the existing Helm 2 configuration.  Always 
   test with the ``--dry-run`` option before migrating or updating a production system.
```

File: docs/docsource/installation/upgrade/upgrade_zenko.rst [hunk 5]
```diff
@@ -87,3 +87,34 @@ To upgrade Zenko:
...
+  test with the --dry-run option before migrating or updating production 
+  systems. 
+
+  ::
+
+     $ helm3 2to3 move config --dry-run
+     $ helm3 2to3 move config
+
+  After the Helm 3 configuration is complete you can convert your existing
+  release with the following steps.
```
Comment (wabernat) [inline docs/docsource/installation/upgrade/upgrade_zenko.rst]:
```suggestion
#. After Helm 3 configuration is complete, convert the existing
    deployment with the following steps:
```

File: docs/docsource/installation/upgrade/upgrade_zenko.rst (hunk 1 above)
Comment (wabernat) [reply docs/docsource/installation/upgrade/upgrade_zenko.rst]: This is not a continuation of the previous process. It introduces a new process (numbered, below)

**Review by mrepop — COMMENTED** [review]
Adding in suggestions from william

**Review by wabernat — APPROVED** [review]

**Review by tmacro — APPROVED** [review]

**Pull request merged**